genai-perf/docs/goodput.md

```diff
@@ -66,6 +66,7 @@ genai-perf profile \
     --service-kind openai \
     --endpoint-type chat \
     --streaming \
+    --request-count 10 \
```

Comment:
Added request-count here since the other example uses measurement interval. This wasn't strictly necessary, but I liked the symmetry of showing both options.